app/views/claims/subjects_taught.html.erb

```diff
@@ -21,9 +21,9 @@
               <% end %>
               <div class="govuk-radios__divider">or</div>
               <div class="govuk-radios__item">
-                <%= fields.hidden_field :mostly_teaching_eligible_subjects, value: '' %>
-                <%= fields.radio_button(:mostly_teaching_eligible_subjects, false, class: "govuk-radios__input")  %>
-                <%= fields.label :mostly_teaching_eligible_subjects_false, t('student_loans.questions.eligible_subjects.not_applicable'), class: "govuk-label govuk-radios__label" %>
+                <%= fields.hidden_field :mostly_teaching, value: '' %>
+                <%= fields.radio_button(:mostly_teaching, false, class: "govuk-radios__input")  %>
+                <%= fields.label :mostly_teaching_false, t('student_loans.questions.eligible_subjects.not_applicable'), class: "govuk-label govuk-radios__label" %>
```

Comment:
This is technically wrong. Just because the user hasn't taught eligible subjects doesn't mean they weren't teaching enough. I've also just realized that that we currently share an eligibility failure reason between this and the case where a teacher did mostly leadrship stuff, which needs to change. I'd like to do this without requiring another database field just for this, but I'm not sure how to do that reliably with `subjects_taught` as an array. Any thoughts?

Reply:
@erbridge can we check with policy? I think they definitely need to be teaching at least one of these subjects to be eligible.

Reply:
oh, just seen what you mean... I'll have a ponder but it might be that this requires an additional database column..

Reply:
Yeah, that's what I'm thinking. Since I'm changing the columns, I'll leave the existing column (with a rename) for now.